benchmark-service/benchmark/benchmark.py

```diff
@@ -200,8 +207,37 @@ async def compare(request, userdata):  # pylint: disable=unused-argument
     return await render_template('benchmark', request, userdata, 'compare.html', context)
 
 
+async def query_github(app):
+    global START_POINT
+    github_client = app['github_client']
+    request_string = f'/repos/hail-is/hail/commits?since={START_POINT}'
+
+    data = await github_client.getitem(request_string)
+    new_commits = []
+    for commit in data:
+        sha = commit.get('sha')
+        new_commits.append(commit)
+        log.info(f'commit {sha}')
+        START_POINT = commit.get('commit').get('author').get('date')
```

Comment:
get will return `None` if the key doesn't exist and then you'll get an error on the subsequent call to `.get`. That error won't include which key caused the error. I think you're bette roff with:
```
commit['commit']['author'].get('date')
```
If you don't expect `date` to be missing, you can use `[]` for that too.